README.md

```diff
@@ -1,17 +1,58 @@
 # Kontrol
 
+Fast Installation
+-----------------
 
-## Installation
+-   `bash <(curl https://kframework.org/install)`: install [kup package manager].
```

Comment:
```suggestion
-   `bash <(curl https://kframework.org/install)`: install [kup package manager](https://github.com/runtimeverification/kup).
```

Reply:
Added the link at the end of the document to avoid duplication.